templates/helm-values-grafana.tpl.yaml

```diff
@@ -0,0 +1,10 @@
+dataSource:  
+  prometheus-datasource.json: |+
```

Comment:
won't this create a duplicate source?

Reply:
Can you elaborate what you mean?

Reply:
currently the infrastructure prometheus is already a datasource. won't setting this create 2 datasources of the same endpoint?

Reply:
No from the moment you create one datasource for helm grafana, you have to declare them all.
See [this](https://github.com/coreos/prometheus-operator/blob/master/helm/grafana/templates/dashboards-configmap.yaml#L16-L27) if..else structure